{{cookiecutter.project_shortname}}/Dockerfile.base

```diff
@@ -4,4 +4,4 @@ FROM inveniosoftware/centos7-python:3.6
 COPY ./ .
 COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
 
-RUN ./scripts/bootstrap --deploy
+RUN ./scripts/bootstrap --deploy --dev --skip-lock --pre
```

Comment:
This could be another way to pass the flags to `pipenv install`. If this is preferred #87 can be closed